src/sidebar/components/ThreadList.tsx

```diff
@@ -209,6 +209,18 @@ export default function ThreadList({ threads }: ThreadListProps) {
     }
   }, [store, newAnnotationTag]);
 
+  const editing = store.countDrafts() > 0;
+  const highlightedAnnotations = store.highlightedAnnotations();
+
+  // Scroll to the first highlighted annotation, unless creating/editing another
+  // annotation
+  useEffect(() => {
+    const [firstHighlightedAnnotation] = highlightedAnnotations;
```

Comment:
This doesn't take account of the current sort order in the thread list. I guess it will scroll to whichever update came in first, if that is the order in which highlights naturally are sorted in the store?

One easy way to make this feel predictable might be to sort by last update time and scroll to the oldest (ie. the first update the user hasn't seen yet) or alternatively the newest.

Reply:
The only intention here was to make sure the user sees at least one of the highlighted annotations, without worrying too much which one is it.

But we could definitely pick one by date, as that should not complicate this too much.